packages/trie/src/trie/trie.ts

```diff
@@ -94,6 +92,18 @@ export class Trie {
     return new Trie(opts)
   }
 
+  database(db?: DB) {
```

Comment:
I'm not sure do we want the feature that one can set the DB after creating a Trie (so put some values, then change DB, and put more values..?). This seems rather error prone since the new DB most likely does not have the right key value pairs?

Reply:
Right, yes, I agree.